### ISSUE TITLE
Proposal: obsolete meso/metanephros subtypes for many kidney cell types

### ISSUE THREAD

**Issue by cmungall** [obophenotype/cell-ontology#512]
From KPMP meeting

E.g. http://purl.obolibrary.org/obo/CL_0000653

![image](https://user-images.githubusercontent.com/50745/43228894-bed3e9c4-9017-11e8-928d-bfa0ee491b5e.png)

This is generally confusing since

 1. the meso form is likely rudimentary / non functional in mammals
 2. better to post-compose these

**Comment by cerivs** [comment]
Meso form would be the form found in fish.

**Comment by cmungall** [comment]
Yep. Do you typically use the pre-composed subclass or annotate to the main cell type? This Q applies to other cell types too. Quick browse of ZP shows a mix but I haven't rigorously checked.

The danger is that some proportion of curators will annotate to cell type C since it's "obvious" what the kidney stage is, in other cases a ZFIN/MGI curator will annotate to "meso-C" or "meta-C", since they have been trained to be as specific as possible, and we end up with a bit of an ad-hoc mix of specificity. Of course we can infer the subtyping if we also have stage info (usually) but this complicates the overall pipeline

**Comment by cerivs** [comment]
We should be making sure our curators have  and use the specific term type.  I would prefer our annotations go to the most specific cell type and have automated processes pull the annotations together rather than requiring users to realize that our annotations are to cell types in different anatomical structures (pronephros, mesonephros and metanephros). I think we need the general cell type for the cross species or cross stages comparison but we might need the specific terms so inherent differences based on  developmental stage or species are simpler to determine.
